src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -136,6 +140,19 @@ public org.folio.des.domain.dto.Job upsertAndSendToKafka(org.folio.des.domain.dt
       result.setExitStatus(ExitStatus.UNKNOWN);
     }
 
+    Optional.ofNullable(jobDto.getExportTypeSpecificParameters())
+      .map(ExportTypeSpecificParameters::getVendorEdiOrdersExportConfig)
+        .map(VendorEdiOrdersExportConfig::getExportConfigId).ifPresent(configId -> {
+          try {
+            log.info("Config with id {} not found", configId.toString());
+            client.getConfigById(configId.toString());
+          } catch (NotFoundException e) {
+            log.info("config not found", configId.toString());
```

Comment:
Add paramter placeholder: So result will be like this: "Config with id {} not found"